azure-pipelines.yml

```diff
@@ -16,27 +16,33 @@ variables:
   backwardCompatibleRelease: '1.0.0'
   forwardCompatibleRelease: '1.0.0'
 
-  backwardCompatibleTestOptions_Windows_2_3: ""
-  forwardCompatibleTestOptions_Windows_2_3: ""
-  backwardCompatibleTestOptions_Linux_2_3: ""
-  forwardCompatibleTestOptions_Linux_2_3: ""
-
-  backwardCompatibleTestOptions_Windows_2_4: ""
-  forwardCompatibleTestOptions_Windows_2_4: ""
-  backwardCompatibleTestOptions_Linux_2_4: ""
+  # Filter Microsoft.Spark.E2ETest.IpcTests.ML.Linalg.SparseVectorTests.TestSparseVectorE2EScenario
+  # Please see https://github.com/dotnet/spark/pull/810
+  backwardCompatiableTestIgnore_PR_810: "FullyQualifiedName!=Microsoft.Spark.E2ETest.IpcTests.ML.Linalg.SparseVectorTests.TestSparseVectorE2EScenario"
+
+  backwardCompatibleTestOptions_Windows_2_3: "--filter ($(backwardCompatiableTestIgnore_PR_810))"
+  forwardCompatibleTestOptions_Windows_2_3: "--filter ($(backwardCompatiableTestIgnore_PR_810))"
```

Comment:
Weird, it shouldn't break the forward compatible tests since this new test shouldn't exist. (we are doing "Checkout dotnet/spark@refs/tags/v1.0.0 to s$(PATH_SEPARATOR)dotnet-spark-1.0.0" and build/test unless it's broken somehwere).

Could you share the log for the failures?

Reply:
I had another look and the forward tests were breaking because maven was getting a reset connection: https://dev.azure.com/dnceng/public/_build/results?buildId=959021&view=logs&j=752a2194-d368-5dec-19f4-d06166099277&t=b757f20e-e579-55f4-086e-3f4cbc799e47

I’ll put them back in and see what happens.

Reply:
I re-triggered two failing tests due to maven, and tests are all passing now.

Reply:
Oh wait, it seems that we still have the filter for forward compatibility tests?

Reply:
I was on my phone when I said i'd change it - have changed it now :).